### DISCUSSION TITLE
Can serverpod mini server side run in flutter?

### DISCUSSION THREAD

**Discussion by bailyzheng** [serverpod/serverpod#3092] [Q&A] [answered]
It is very useful for IoT that run a mini server in flutter. Is there any plan to support?

**Comment by SandPod** [comment, marked as answer]
Hey @bailyzheng!

I think it should already be possible to run Serverpod on any device that supports dart:io.
Therefore it should be possible to run Serverpod as long as your are not running it as a web app.

**Reply by SandPod** [reply]
If you encountered any issues, could you elaborate on what you encountered?
